admin/app/views/spree/admin/page_blocks/forms/_buttons.html.erb

```diff
@@ -1,13 +1,13 @@
 <%= render 'spree/admin/page_blocks/forms/link', f: f %>
 
 <div class="form-group">
-  <%= f.label :preferred_button_style_1 %>
+  <%= f.label :preferred_button_style_1, Spree.t(:button_style) %>
   <%= f.select :preferred_button_style_1, options_for_select(['primary', 'secondary'], @page_block.preferred_button_style_1), {  }, { class: 'custom-select', data: {  action: 'auto-submit#submit' } } %>
 </div>
 
 <% content_for(:design_tab) do %>
   <div class="form-group">
     <%= f.label "Buttons alignment" %>
-    <%= f.select :preferred_buttons_alignment, options_for_select(['left', 'center', 'right'], @page_block.preferred_buttons_alignment), {}, { class: 'custom-select', data: {  action: 'auto-submit#submit' } } %>
```

Comment:
@mad-eel should this be text alignment if the label is button alignment? also, the label should be using translations :)

Reply:
@damianlegawiec I added locale here and updated the `config/locales/en.yml` file with all missing locales